### DISCUSSION TITLE
Hair (TressFX) crashes in Debug configuration

### DISCUSSION THREAD

**Discussion by ohmaya** [o3de/o3de#11358] [Help Wanted]
1. Build the project in Debug configuration.
2. Create an entity.
3. Add an Actor component, assign \o3de\Gems\AtomTressFX\Assets\TestData\HeadBoneChainHairStyle\test_hair_bone_chain_head_style.fbx.
4. Add an Ator Hair (TressFX) component, assign \o3de\Gems\AtomTressFX\Assets\TestData\HeadBoneChainHairStyle\test_hair_bone_chain_head_style.tfx.

Crashes in CommandList.cpp > Validation layer.

**Comment by santorac** [comment]
I recommend reporting this as GitHub Issue rather than a Discussion. https://github.com/o3de/o3de/issues/new/choose
